### PR TITLE
chore(license): update license for 1.2

### PR DESCRIPTION
**Pull request by liuxiaocs7**
## Purpose of the PR

- refer: https://github.com/hugegraph/actions/issues/6

## Verifying these changes

- [ ] Trivial rework / code cleanup without any test coverage. (No Need)
- [ ] Already covered by existing tests, such as *(please modify tests here)*.
- [ ] Need tests and can be verified as follows:
    - xxx

## Does this PR potentially affect the following parts?

- [x]  Nope
- [ ]  Dependencies (add/update license info) 
- [ ]  Modify configurations
- [ ]  The public API
- [ ]  Other affects (typed here)

## Documentation Status

- [ ]  `Doc - TODO` 
- [ ]  `Doc - Done` 
- [x]  `Doc - No Need`

### CODE REVIEW

File: hugegraph-dist/release-docs/NOTICE [hunk 1]
```diff
@@ -2395,29 +2254,55 @@ source code repository logs.
...
+ASM 7 (n/a)
+
+* License: BSD-3-Clause
+* Project: https://asm.ow2.io/
+* Source:
+   https://repository.ow2.org/nexus/#nexus-search;gav~org.ow2.asm~asm-commons~~~~kw,versionexpand
+
+JTHarness (5.0)
+
+* License: (GPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0)
```
Comment (imbajin) [inline hugegraph-dist/release-docs/NOTICE:2286]: where does it come from? `*GPL*` is not allowed in LICENSE & NOTICE normally

https://www.apache.org/legal/resolved.html#category-x

File: hugegraph-dist/release-docs/NOTICE [hunk 2]
```diff
@@ -2395,29 +2254,55 @@ source code repository logs.
...
+* Source: http://hg.openjdk.java.net/code-tools/jtharness/
+
+normalize.css (3.0.2)
 
-None
+* License: MIT
 
+SigTest (n/a)
+
+* License: GPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
```
Comment (imbajin) [inline hugegraph-dist/release-docs/NOTICE:2296]: same

File: hugegraph-dist/release-docs/NOTICE [hunk 3]
```diff
@@ -2468,3 +2353,408 @@ package in the Spring Framework library, distributed by VMware, Inc:
...
+
+This program and the accompanying materials are made available under the terms
+of the Eclipse Public License v. 2.0 which is available at
+http://www.eclipse.org/legal/epl-2.0. This Source Code may also be made
+available under the following Secondary Licenses when the conditions for such
+availability set forth in the Eclipse Public License v. 2.0 are satisfied: GNU
+General Public License, version 2 with the GNU Classpath Exception which is
+available at https://www.gnu.org/software/classpath/license.html.
+
+SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
```
Comment (imbajin) [inline hugegraph-dist/release-docs/NOTICE:2387]: same

**Review by imbajin — CHANGES_REQUESTED** [review]

File: hugegraph-dist/release-docs/NOTICE (hunk 1 above)
Comment (liuxiaocs7) [reply hugegraph-dist/release-docs/NOTICE:2286]: Jakarta Activation API
https://github.com/jakartaee/jaf-api/blob/master/NOTICE.md

File: hugegraph-dist/release-docs/NOTICE (hunk 3 above)
Comment (liuxiaocs7) [reply hugegraph-dist/release-docs/NOTICE:2289]: Do I need to remove it manually?

https://github.com/apache/hadoop/blob/62cc673d00ed64437fc787f85c8091f8357461da/NOTICE-binary#L460-L466

File: hugegraph-dist/release-docs/NOTICE (hunk 1 above)
Comment (liuxiaocs7) [reply hugegraph-dist/release-docs/NOTICE]: removed

**Review by imbajin — APPROVED** [review]
merge it first, check it in local IDE soon

**Review by simon824 — APPROVED** [review]

**Pull request merged**